tests/functional/raw-node/test/GCP/object/completeMpu.js

```diff
@@ -0,0 +1,203 @@
+const assert = require('assert');
+const async = require('async');
+const { GCP, GcpUtils } = require('../../../../../../lib/data/external/GCP');
+const { gcpRequestRetry, setBucketClass, gcpMpuSetup } =
+    require('../../../utils/gcpUtils');
+const { getRealAwsConfig } =
+    require('../../../../aws-node-sdk/test/support/awsConfig');
+
+const credentialOne = 'gcpbackend';
+const bucketNames = {
+    main: {
+        Name: `somebucket-${Date.now()}`,
+        Type: 'MULTI_REGIONAL',
+    },
+    mpu: {
+        Name: `mpubucket-${Date.now()}`,
+        Type: 'REGIONAL',
+    },
+    overflow: {
+        Name: `overflowbucket-${Date.now()}`,
+        Type: 'MULTI_REGIONAL',
+    },
+};
+const numParts = 10000;
+const partSize = 10;
+
+const smallMD5 = '583c466f3f31d97b361adc60caea72f5-1';
+const bigMD5 = '0bd3785d5d1e3c90988917837bbf57fc-10000';
+
+function gcpMpuSetupWrapper(params, callback) {
+    gcpMpuSetup(params, (err, result) => {
+        assert.equal(err, null,
```

Comment:
You can also use `assert.ifError(err, ...)`

Reply:
Thanks! simpler/more descriptive